projects/plugins/jetpack/_inc/client/at-a-glance/backups.jsx

```diff
@@ -215,9 +208,9 @@ class DashBackups extends Component {
 			} );
 		}
 
-		if ( ! isEmpty( sitePlan ) ) {
+		if ( ! isFetchingSite ) {
 			// If site has a paid plan
-			if ( 'jetpack_free' !== get( sitePlan, 'product_slug', 'jetpack_free' ) ) {
+			if ( hasVaultPressBackups ) {
```

Comment:
@vianasw The new VaultPress Backups feature is currently associated with JP premium and business plans (2c4c3-pb/#plain), which doesn't seem to line up with this conditional of any paid plan. Should it be premium and higher plans (2c4c4-pb/#plain) or truly any paid plan?

(If you're not the best person to ask, could you point me to who would be?)

Reply:
I think personal and higher is a better fit as Personal (a now legacy plan) also had backups.

Reply:
Looking further here, I'm not sure that there needs to be any differentiation between VaultPress backups and Jetpack Backups. 

The issue as I understand it is more of a technical one, not one driven by whether a plan has been purchased or not.

That being said, there should be no issue keeping the VaultPress Backups feature check.

 @vianasw @jwebbdev - Will one of you confirm this please?

Reply:
Agreed, no distinction is required between VaultPress and Jetpack for Jetpack plans.  As far as I know all paid **bundle** plans include backup, then there are also the individual backup product plans.